packages/core/core.ts

```diff
@@ -124,11 +127,50 @@ export function startFuzzingAsyncNoInit(
 	return Fuzzer.startFuzzingAsync(fuzzFn, fuzzerOptions);
 }
 
-export function stopFuzzingAsync() {
+export function stopFuzzing(err: unknown, expectedErrors: string[]) {
+	// No error found, check if one is expected.
+	if (!err) {
+		if (expectedErrors.length) {
+			console.error(
+				`ERROR: Received no error, but expected one of [${expectedErrors}].`
+			);
+			Fuzzer.stopFuzzingAsync(ERROR_UNEXPECTED_CODE);
+		}
+		return;
+	}
+
+	// Error found and expected, check if it's the correct one.
```

Comment:
It's already in the code but strictly speaking, *the correct one* is misleading as it checks for `actualErr ∈ expectedErrList`

Reply:
Changed the comment.